crates/ruff_workspace/src/configuration.rs

```diff
@@ -1454,9 +1460,14 @@ fn warn_about_deprecated_top_level_lint_options(
         .map(|option| format!("- '{option}' -> 'lint.{option}'"))
         .join("\n  ");
 
+    let thing_to_update = path.map_or_else(
+        || String::from("your `--config` CLI arguments"),
+        |path| format!("`{}`", fs::relativize_path(path)),
+    );
```

Comment:
Is there test coverage for this?

Reply:
There is now :)

7d6f9dd1216f7160e95e6d6d999d6cc32bcc21d5